samples/v1/types_config.go

```diff
@@ -64,6 +64,14 @@ type ConfigSpec struct {
 	// content but the operator will not recreate(or update) anything
 	// listed here.
 	SkippedTemplates []string `json:"skippedTemplates,omitempty" protobuf:"bytes,6,opt,name=skippedTemplates"`
+
+	// skippedhelmcharts specifies names of helmcharts that should NOT be
+	// created/updated.  Admins can use this to allow them to delete content
+	// they don’t want.  They will still have to manually delete the
+	// content but the operator will not recreate(or update) anything
+	// listed here.
+	// +listType=set
+	SkippedHelmCharts []string `json:"skippedHelmCharts,omitempty" protobuf:"bytes,7,opt,name=skippedhelmCharts"`
```

Comment:
I had initially kept a limit of 12, as the new changes would require operator to install 12 helmcharts.. but not sure about the future if we need to increase it and hence got a suggestion to remove the validation of 12 

> > ```
> >  I Kept the validation as 12 max items because there are 12 S2i currently which we handle
> > ```
> 
> yeah i assumed that was your logic, but it means you'll need to update this limit if you ever add more, and i don't really see that the having the validation/constraint in place is worth that cost.

Reply:
How about a limit of 32? That's small enough that it stops someone going crazy with the field, but large enough that there's overhead from existing know use cases.

Or if you're confident, we could go for 16. Loosening the validation later is in theory easy assuming we control the consumers, which I think is just the samples operator right?

Reply:
Its just Samples Operator, and we are downsizing, so mostly 16 should suffice in my opinion. i will update it with max allowed as 16 .